src/components/Navigation.js

```diff
@@ -24,9 +24,12 @@ const Navigation = () => {
           <Link to="/addInmate">Add an Inmate </Link>
         </Menu.Item>
         <Menu.Item key="5">
-          <Link to="/login">Login </Link>
+          <Link to="/addPrison">Add a Prison </Link>
```

Comment:
This is okay for now to access the form but we'll need to have this link siting in the facilities page.